kolibri/core/assets/src/views/__tests__/UserTypeDisplay.spec.js

```diff
@@ -0,0 +1,45 @@
+import VueRouter from 'vue-router';
+import { render, screen } from '@testing-library/vue';
+import UserTypeDisplay from '../UserTypeDisplay.vue';
+
+const sampleUserType = 'testing-user-type';
+const translatedSampleUserType = 'Testing User Type';
```

Comment:
I think we should change `translated` here (and throughout the file) to `expected`. When I read `translate` I started thinking this was testing something around localization 😅

Reply:
Got it!